### PR TITLE
Day one

### PR DESCRIPTION
**Pull request by Qairkhan**
The main components of the site are added. Used grids. All components are divided into subcomponents and divided into folders. Used by jsx and .component.css 
Next step: add other pages

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
```
Comment (Dmitry-White) [inline src/components/Footer/Footer.js:2]: Consider removing redundant empty lines.
The rule is to have only one empty line that separates blocks of grouped code.

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    return (
+        <div>
+            <a href='#'>2</a>
+            <a href='#'>2</a>
+            <a href='#'>2</a>
+            <a href='#'>2</a>
+
+        </div>
+    );
+
```
Comment (Dmitry-White) [inline src/components/Footer/Footer.js]: Consider removing a redundant empty line.

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+const Footer = () => {
+    return (
+        <div>
+            <a href='#'>2</a>
+            <a href='#'>2</a>
+            <a href='#'>2</a>
+            <a href='#'>2</a>
+
```
Comment (Dmitry-White) [inline src/components/Footer/Footer.js]: Consider removing a redundant empty line

File: src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import s from "./Header.module.css";
+
+
```
Comment (Dmitry-White) [inline src/components/Header/Header.jsx:4]: Consider removing redundant empty lines.
The rule is to have only one empty line that separates blocks of grouped code.

File: src/components/Header/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+const Header = () => {
+    return (
+        <header className = {s.header}>
+        <img src="https://i.pinimg.com/originals/d9/12/09/d91209340bdc005936c46323a62caaff.png"/>
+      </header>
+    );
+
```
Comment (Dmitry-White) [inline src/components/Header/Header.jsx]: Consider removing a redundant empty line.

File: src/components/Header/Header.module.css [hunk 6]
```diff
@@ -0,0 +1,9 @@
+.header {
+  grid-area: h;
+  background-color: gray;
+
```
Comment (Dmitry-White) [inline src/components/Header/Header.module.css]: Consider removing a redundant empty line.

File: src/components/Navbar/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import s from "./Navbar.module.css";
+
+
```
Comment (Dmitry-White) [inline src/components/Navbar/Navbar.jsx:4]: Consider removing redundant empty lines.
The rule is to have only one empty line that separates blocks of grouped code.

File: src/components/Navbar/Navbar.module.css [hunk 8]
```diff
@@ -0,0 +1,10 @@
+.nav {
+  grid-area: n;
+  background-color: aqua;
+
```
Comment (Dmitry-White) [inline src/components/Navbar/Navbar.module.css]: Consider removing redundant empty lines.

File: src/components/Profile/MyPosts/Post/Post.module.css [hunk 9]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (Dmitry-White) [inline src/components/Profile/MyPosts/Post/Post.module.css]: Consider removing redundant empty lines.

File: src/components/Profile/Profile.module.css [hunk 10]
```diff
@@ -0,0 +1,13 @@
+
```
Comment (Dmitry-White) [inline src/components/Profile/Profile.module.css]: Consider removing redundant empty lines.

File: src/App.js [hunk 11]
```diff
@@ -1,24 +1,18 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```
Comment (Dmitry-White) [inline src/App.js:1]: Consider separating third-party imports from the project's ones.
So first imports in a file should be third-party imports like "react".
Then your project common imports that appear in almost all the components - like for example "../../utils/your-util.js".
Then your component related stuff - like "./your-component-helper.js".
And then the css imports.

Every import group should be separated from other import groups with an empty new line.

Don't forget to do this in all other files of the project.

File: src/components/Header/Header.jsx [hunk 12]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import s from "./Header.module.css";
```
Comment (Dmitry-White) [inline src/components/Header/Header.jsx]: "s" is hard to read in terms of understanding what this variable is for.
Consider using a full name such as "styles".

Don't forget to do this in all other file of the project.

File: src/components/Header/Header.jsx [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+
+
+
+const Header = () => {
+    return (
+        <header className = {s.header}>
+        <img src="https://i.pinimg.com/originals/d9/12/09/d91209340bdc005936c46323a62caaff.png"/>
```
Comment (Dmitry-White) [inline src/components/Header/Header.jsx]: Consider putting this string url into a constant variable in the current file and using the variable reference here.

File: src/components/Profile/MyPosts/Post/Post.jsx [hunk 14]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import s from "./Post.module.css";
+
+const Post = () => {
+  return (
+    <div className={s.item}>
+      <img src="https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/86/86ee5650444ed1a82866aea7e6ce7e86aa3d560d_full.jpg" />
```
Comment (Dmitry-White) [inline src/components/Profile/MyPosts/Post/Post.jsx]: Consider putting this string url into a constant variable in the current file and using the variable reference here.

File: src/components/Profile/Profile.jsx [hunk 15]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import s from "./Profile.module.css";
+import MyPosts from "./MyPosts/MyPosts";
+
+const Profile = () => {
+  return (
+    <div className={s.content}>
+      <div>
+        <img src="https://animal-wallpaper.com/wallpaper/minimalist-background-hd-For-Background-HD-Wallpaper.jpg" />
```
Comment (Dmitry-White) [inline src/components/Profile/Profile.jsx]: Consider putting this string url into a constant variable in the current file and using the variable reference here.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js (hunk 2 above)
Comment (Qairkhan) [reply src/components/Footer/Footer.js]: fixed

File: src/components/Header/Header.jsx (hunk 4 above)
Comment (Qairkhan) [reply src/components/Header/Header.jsx:4]: fixed

File: src/components/Header/Header.jsx (hunk 5 above)
Comment (Qairkhan) [reply src/components/Header/Header.jsx]: fixed

File: src/components/Header/Header.module.css (hunk 6 above)
Comment (Qairkhan) [reply src/components/Header/Header.module.css]: fixed

File: src/components/Navbar/Navbar.jsx (hunk 7 above)
Comment (Qairkhan) [reply src/components/Navbar/Navbar.jsx:4]: fixed

File: src/components/Navbar/Navbar.module.css (hunk 8 above)
Comment (Qairkhan) [reply src/components/Navbar/Navbar.module.css]: fixed

File: src/components/Profile/MyPosts/Post/Post.module.css (hunk 9 above)
Comment (Qairkhan) [reply src/components/Profile/MyPosts/Post/Post.module.css]: fixed

File: src/components/Profile/Profile.module.css (hunk 10 above)
Comment (Qairkhan) [reply src/components/Profile/Profile.module.css]: fiixed

File: src/App.js (hunk 11 above)
Comment (Qairkhan) [reply src/App.js:1]:
```
import React from "react";


import Header from "./components/Header/Header";
import Navbar from "./components/Navbar/Navbar";
import Profile from "./components/Profile/Profile";

import "./App.css";
```

like so?

File: src/components/Header/Header.jsx (hunk 12 above)
Comment (Qairkhan) [reply src/components/Header/Header.jsx]: "s" is used for shorter spelling in tags. will be fixed in the future

File: src/components/Header/Header.jsx (hunk 13 above)
Comment (Qairkhan) [reply src/components/Header/Header.jsx]: complite

File: src/components/Profile/MyPosts/Post/Post.jsx (hunk 14 above)
Comment (Qairkhan) [reply src/components/Profile/MyPosts/Post/Post.jsx]: complite

File: src/components/Profile/Profile.jsx (hunk 15 above)
Comment (Qairkhan) [reply src/components/Profile/Profile.jsx]: complite

File: src/components/Header/Header.jsx [hunk 16]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import s from "./Header.module.css";
+
+const HeaderLink = "https://i.pinimg.com/originals/d9/12/09/d91209340bdc005936c46323a62caaff.png"
```
Comment (Dmitry-White) [inline src/components/Header/Header.jsx]: Consider changing the case of "HeaderLink" to upper case like so "HEADER_LINK".
For your information:
1. "YOUR_NAME" - such naming indicates the constants (not the "const" variables that do not change in the CURRENT SCOPE, but actual constants that will not change for the ENTIRE project)
2. "YourName" - such naming indicates classes, constructor functions (everything that is used with "new YourName()) and components (like React components)
3. "yourName" - such naming is used for variables and regular functions.

File: src/components/Header/Header.jsx [hunk 17]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
```
Comment (Dmitry-White) [inline src/components/Header/Header.jsx:1]: Consider adding an empty line between a third-party imports and your css.

File: src/components/Navbar/Navbar.jsx [hunk 18]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
```
Comment (Dmitry-White) [inline src/components/Navbar/Navbar.jsx:1]: Consider adding an empty line between a third-party imports and your css.

File: src/components/Profile/MyPosts/Post/Post.jsx [hunk 19]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
```
Comment (Dmitry-White) [inline src/components/Profile/MyPosts/Post/Post.jsx:1]: Consider adding an empty line between a third-party imports and your css.

File: src/components/Profile/MyPosts/Post/Post.jsx [hunk 20]
```diff
@@ -0,0 +1,15 @@
...
+import s from "./Post.module.css";
+
+const AvatartsLink = "https://steamcdn-a.akamaihd.net/steamcommunity/public/images/avatars/86/86ee5650444ed1a82866aea7e6ce7e86aa3d560d_full.jpg"
```
Comment (Dmitry-White) [inline src/components/Profile/MyPosts/Post/Post.jsx]: Consider changing the case of "AvatarLink" to upper case like so "AVATAR_LINK".
Check the spelling, mind the typing mistakes.

File: src/components/Profile/Profile.jsx [hunk 21]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import s from "./Profile.module.css";
+import MyPosts from "./MyPosts/MyPosts";
+
+const ProfileLink = "https://animal-wallpaper.com/wallpaper/minimalist-background-hd-For-Background-HD-Wallpaper.jpg"
```
Comment (Dmitry-White) [inline src/components/Profile/Profile.jsx]: Consider changing the case of "ProfileLink" to upper case like so "PROFILE_LINK".

File: src/components/Header/Header.jsx (hunk 12 above)
Comment (Dmitry-White) [reply src/components/Header/Header.jsx]: Fine

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/components/Header/Header.jsx (hunk 16 above)
Comment (Qairkhan) [reply src/components/Header/Header.jsx]: complite

File: src/components/Navbar/Navbar.jsx (hunk 18 above)
Comment (Qairkhan) [reply src/components/Navbar/Navbar.jsx:1]: complite

File: src/components/Header/Header.jsx (hunk 17 above)
Comment (Qairkhan) [reply src/components/Header/Header.jsx:1]: complite

File: src/components/Profile/MyPosts/Post/Post.jsx (hunk 19 above)
Comment (Qairkhan) [reply src/components/Profile/MyPosts/Post/Post.jsx:1]: complite

File: src/components/Profile/Profile.jsx (hunk 21 above)
Comment (Qairkhan) [reply src/components/Profile/Profile.jsx]: complite

**Review by Dmitry-White — APPROVED** [review]
Nice work!

**Pull request merged**